### ISSUE TITLE
boxsdk 1.x does not import in python 3.6.0

### ISSUE THREAD

**Issue by incidentist** [box/box-python-sdk#195]
Running a simple script in Python 3.6.0 fails to import the boxsdk module:

```> python3 sources/box.py 
Traceback (most recent call last):
  File "sources/box.py", line 1, in <module>
    from boxsdk import OAuth2, Client
  File "/usr/local/lib/python3.6/site-packages/boxsdk/__init__.py", line 6, in <module>
    from .client import *  # pylint:disable=wildcard-import,redefined-builtin
  File "/usr/local/lib/python3.6/site-packages/boxsdk/client/__init__.py", line 5, in <module>
    from .client import Client
  File "/usr/local/lib/python3.6/site-packages/boxsdk/client/client.py", line 12, in <module>
    from ..object.events import Events
  File "/usr/local/lib/python3.6/site-packages/boxsdk/object/events.py", line 15, in <module>
    class EventsStreamType(with_metaclass(ExtendableEnumMeta, TextEnum)):
  File "/usr/local/Cellar/python3/3.6.0/Frameworks/Python.framework/Versions/3.6/lib/python3.6/enum.py", line 119, in __prepare__
    member_type, first_enum = metacls._get_mixins_(bases)
  File "/usr/local/Cellar/python3/3.6.0/Frameworks/Python.framework/Versions/3.6/lib/python3.6/enum.py", line 439, in _get_mixins_
    raise TypeError("new enumerations must be created as "
TypeError: new enumerations must be created as `ClassName([mixin_type,] enum_type)`
```

This is on OS X Sierra, with the latest stable version (1.5.3). The script works fine in Python 3.5.2. Tox tests also fail on the 1.5.3 tag, but run cleanly on HEAD.

**Comment by jmoldow** [comment]
This is a bug in `six`. Though we did remediate it in this repo, in PR #183. But it is currently only available in the alpha version >=2.0.0a3.

We haven't yet backported any code from 2.0.0a back to 1.x, but we could do that if necessary.

**Comment by incidentist** [comment]
I think that would be wise, since this is an instant hard crash with the latest stable version. For now, specifying `boxsdk==2.0.0a4` in my pip requirements.txt does what I need it to do.

**Comment by jmoldow** [comment]
Filed https://bitbucket.org/gutworth/six/issues/168/with_metaclasss-temporary-metaclass-doesnt

**Comment by Jeff-Meadows** [comment]
@jmoldow the repo owner closed your issue - I think he'd like you to file it on GitHub instead.

**Comment by jmoldow** [comment]
Yeah, I re-filed it as a Github PR: https://github.com/benjaminp/six/pull/178

**Comment by prahladyeri** [comment]
The `boxsdk==2.0.0a4` workaround suggested by @incidentist works great, but this needs to be solved at priority.

**Comment by jmoldow** [comment]
This was fixed version 1.5.4 (pull request #217)
